app/models/family.rb

```diff
@@ -229,6 +229,9 @@ def self.mappable_details
       "where people.visible = #{Family.connection.quoted_true} " \
       "and families.visible = #{Family.connection.quoted_true} " \
       "and families.deleted = #{Family.connection.quoted_false} " \
+      "and (people.share_address = TRUE " \
+      "  or  #{Person.logged_in.admin?(:view_hidden_profiles)} " \
+      "  or  people.id = #{Person.logged_in.id} )" \
```

Comment:
We'll have to change the `\` to `+` on the line before also.

Let me know if this works!